### DISCUSSION TITLE
Received an email when applying for the github Copilot student package

### DISCUSSION THREAD

**Discussion by SCZ1174** [community/community#75046] [GitHub Education]
### Hi everyone,

Hello everyone, 
I got an email when I applied for the github Copilot student pack. What does this mean?  I have two emails bound to my github account, and I used the "edu" email address to apply, but my other email address received the email.
![image](https://github.com/community/community/assets/86552551/10e1fe2f-a495-4d24-af30-c1c817c704cf)

Then my Copilot interface looks like this.
![image](https://github.com/community/community/assets/86552551/5fa680ef-8bbb-43cb-9a51-8420b5260c2c)

It still shows "Approved" in "GitHub benefits". (The first rejection was because I didn't fill in all the information)
![image](https://github.com/community/community/assets/86552551/63719d09-aa4f-4eb2-89e5-986c11c6cf08)

**Comment by resslr** [comment]
It says due to "US Sanctions" unfortunately it would seem the country you currently live in is not eligible due to a political matter, I am very sorry to see this happen, hopefully in the future it can be resolved.

**Reply by resslr** [reply]
It must be a glitch that is says your approved and not, my honest suggestion is to wait it out and see if you get the benefits if not, then it is the sanctions.

**Reply by SCZ1174** [reply]
Thank you very much for your reply. I looked up the sanctioned countries. But I don't belong to any of the following countries, do I need to submit a Appeal Form?
![image](https://github.com/community/community/assets/86552551/bba75f82-9487-4a58-a1b3-3957ffdc3566)

**Reply by resslr** [reply]
Yes

**Reply by aaravss1** [reply]
Yes
